src/PageLayout/Notification/Recurring/RecurringDeliveryRequest1.js

```diff
@@ -0,0 +1,97 @@
+                                <p className="donating-recieving" id="receivingContactPhone">{this.props.details.receivingAgency.primaryContact.phone}</p>
+                                <p className="donating-recieving" id="receivingContactEmail">{this.props.details.receivingAgency.primaryContact.email}</p>
+                            </div>
+                            <div className="notification-map">
+                                <Map 
+                                    address={this.props.details.receivingAgency.address} 
+                                    height={'150px'}
+                                    width={'350px'}
+                                    marginRight={'30px'}
+                                    marginTop={'0px'}
+                                    marginBottom={'0px'}
+                                    marginLeft={'10px'}
+                                    className="map"></Map>            
+                            </div> 
+                            <div className="popup-buttons">
+                                <button onClick={this.props.nextStep} className="claim" type="button">Claim</button> 
+                                <button onClick={this.props.close} className="reject" type="button">Reject</button>
+                            </div>
+                        </div>
+                }
+                {
+                    this.props.accountType === AccountType.RECEIVING_AGENCY &&
+                        <div className="receiving-agency">
+                            <h2>Deliverers</h2>
+                            <div className="modal-deliverers">
+                                {
+                                    this.props.details.delivererGroup.deliverers.length === 0 ?
+                                        <p>To be determined</p>
+                                        :
+                                        this.props.details.delivererGroup.deliverers.map((d, i) => {
+                                            return (
+                                                <div className="mobile-student-info" id={i} key={i}>
+                                                    <p id="mobile-name">{d.name}</p>
+                                                    <p id="mobile-org">{d.email}</p>
+                                                    <a href={'tel:' + d.phone}>{d.phone}</a>
+                                                </div>
+                                            );
+                                        })
+                                }
+                            </div>
+                            <div className="popup-buttons">
+                                <button onClick={this.props.enterPrimaryContact} className="claim" type="button">Claim</button> 
+                                <button onClick={this.props.close} className="reject" type="button">Reject</button>
```

Comment:
reject button should go to a local `reject()` function first where backend will need to handle things to do when rejected (eg remove notification from account, etc), and then call `this.props.close`